### PR TITLE
Blog post welcoming Tania as Labs Co-Director

### PR DESCRIPTION
**Pull request by rgommers**
Also update the team gallery and About page. Cc @trallard

### CODE REVIEW

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+is an important step in the growth of Labs - in the past two years it grew
+from an idea to an open source lab [with over 30 members](/team) who
+contribute to key projects in the PyData ecosystem and beyond. Now is the
+right time to strengthen the leadership capability of Labs. I already found
+Tania and I are both very interested in and on the same wavelength about
+community building. That's also how we met in person for the first time,
+Tania and I were speaking back to back in the Open Source Communities track
+at SciPy'19. I'm looking forward to lots of interesting conversations about
+everything from governance in open source projects to diversity, equity and
+inclusion (DEI) to funding. Technically we have complementary strengths and
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:39]:
```suggestion
inclusion (DEI) to funding. We have complementary technological strengths and
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Tania and I will be working closely together and will jointly lead Labs. This
+is an important step in the growth of Labs - in the past two years it grew
+from an idea to an open source lab [with over 30 members](/team) who
+contribute to key projects in the PyData ecosystem and beyond. Now is the
+right time to strengthen the leadership capability of Labs. I already found
+Tania and I are both very interested in and on the same wavelength about
+community building. That's also how we met in person for the first time,
+Tania and I were speaking back to back in the Open Source Communities track
+at SciPy'19. I'm looking forward to lots of interesting conversations about
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:37]:
```suggestion
at SciPy'19. I'm looking forward to many interesting conversations about
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+everything from governance in open source projects to diversity, equity and
+inclusion (DEI) to funding. Technically we have complementary strengths and
+interests. Tania will be taking the lead on DevOps, visualization and IDEs,
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:40]:
```suggestion
interests. Tania will be taking the lead on DevOps, visualization and IDEs
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+while I will focus on numerical topics.
+
+## Where Labs is today
+
+Labs has a clear mission and vision, and an open source contribution, funding
+and employment model that I think is both unique and sustainable (see
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:46]:
```suggestion
Labs has a clear mission and vision, and a model for open source contribution, funding
and employment that I think is both unique and sustainable (see
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+[About](/about) for more). I'm particularly proud of the team we've built,
+with both established maintainers and new technical talent, as well as a
+strong focus on roles other than software development and on DEI. Our team
+includes people from 12 countries in 4 continents, and we contribute to many
+of the projects at the core of the PyData ecosystem.
+
+![Logos of NumPy, Spyder, JupyterLab and Spyder](/images/numpy_scipy_jlab_spyder_logos.png)
+
+The established projects we have contributed to the most in 2020 are NumPy,
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:55]:
```suggestion
I would particularly like to mention our contributions in 2020 to NumPy,
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+Spyder, JupyterLab and SciPy. Contributions included a healthy mix of
+maintenance and innovation. For NumPy, Spyder and JupyterLab we have 5
+maintainers or contributors each, and funding that's continuing in 2021 -
+from `numpy.f2py` and documentation improvements to real-time collaboration
+in JupyterLab and completing Spyder 5 with a new plugin architecture.
+
+![Logos of Data APIs, Ibis and PyData Sparse](/images/dataapis_ibis_sparse.png)
+
+Our innovation activities this year included pushing Ibis and PyData Sparse
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:64]:
```suggestion
Our innovation activities this year also included pushing Ibis and PyData Sparse
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+forward, two projects with potential that fill holes in the dataframe and
+array library landscape. Most important though was the creation of the
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:66]:
```suggestion
array library landscape. 

I would especially like to call out the creation of the
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+[Consortium for Python Data API Standards](https://data-apis.org/) in
+mid-2020. This consortium brings together many array, tensor and dataframe
+library maintainers and has the ambitious aim to tackle the problem of
+fragmentation of Python array and dataframe libraries. It published its first
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:70]:
```suggestion
fragmentation of Python array and dataframe libraries. We published our first
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+RFC for an array API standard in November, and there's more to come in 2021.
+
+## What's next?
+
+Now that we've got an established model and a set of major projects that are
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:75]:
```suggestion
Now that we have an established model and a set of major projects that are
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+well-defined and continuing in 2021, we can focus more on our diverse set of
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:76]:
```suggestion
well-defined and continuing in 2021, we can focus more on a diverse set of
```

File: posts/2021/01/welcome-tania-allard-as-labs-codirector.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+smaller activities in addition to growing the team and organization. There's
+some new projects in the pipeline already that we'll be able to announce in
+the coming weeks and months. I'm sure Tania will also inject her own ideas
```
Comment (mattip) [inline posts/2021/01/welcome-tania-allard-as-labs-codirector.md:79]:
```suggestion
smaller activities in addition to growing the team and organization. New
projects in the pipeline will be announced in the coming weeks and months, 
be sure to watch this space. Tania will also inject her own ideas
```

**Review by mattip — COMMENTED** [review]
Nice post. I suggested some tweaks.

**Comment by trallard** [comment]
LGMT Happy to have this merged ✨

**Comment by rgommers** [comment]
Great, thanks Tania and Matti!

@trallard, the blog post is up at https://labs.quansight.org/blog/2021/01/welcoming-tania-allard-labs-codirector/.

**Pull request merged**